### PR TITLE
Cool Stephan Changes

### PR DESCRIPTION
**Pull request by Lakshay983**
Github demo PR

### CODE REVIEW

**Review by LeftistTachyon — APPROVED** [review]
lgtm

File: Chassis/Chassis.kicad_pro [hunk 1]
```diff
@@ -3,26 +3,171 @@
     "3dviewports": [],
     "design_settings": {
       "defaults": {
-        "board_outline_line_width": 0.1,
-        "copper_line_width": 0.2,
+        "board_outline_line_width": 0.09999999999999999,
+        "copper_line_width": 0.19999999999999998,
```
Comment (LeftistTachyon) [inline Chassis/Chassis.kicad_pro:7]: what's going on here?

Comment (Lakshay983) [reply Chassis/Chassis.kicad_pro:7]: rEALLY COOL STUFF

Comment (LeftistTachyon) [reply Chassis/Chassis.kicad_pro:7]: DOPE ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️ ‼️

**Pull request closed without merging**